dumper.py

```diff
@@ -1,19 +1,36 @@
 #!/usr/local/bin/python3
-
-from redmine.redmine_dumper import RedmineDumper
+import logging
 import argparse
+from redmine.redmine_dumper import RedmineDumper
 
 
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
         '--source', help="Source from where ticket will be dumped. Possible values: (redmine|bugzilla). \
             Default: redmine", default='redmine')
-    parser.add_argument('--project', help="Project name", required=True)
-
+    parser.add_argument('--database', help="SQLite file path", required=True)
+    parser.add_argument('--project', help="Project name")
+    parser.add_argument('-v', action='store_true', help='increase output verbosity')
+    parser.add_argument('-vv', action='store_true', help='extra output verbosity')
     args = parser.parse_args()
+
+    if args.vv is True:
+        logging_level = logging.DEBUG
+    else:
+        if args.v is True:
+            logging_level = logging.INFO
+        else:
+            logging_level = logging.WARNING
```

Comment:
Shouldnt this be default to `INFO`? Why do you we exclude `INFO` from the normal flow of the logs?

Reply:
done, reduced the complexity